test/test_wfmanager_task.py

```diff
@@ -109,7 +149,7 @@ def test_load_failure(self):
                 mock.patch(ERROR_PATH) as mock_error, \
                 mock.patch(WORKFLOW_READER_PATH) as mock_reader:
             mock_dialog.side_effect = mock_file_dialog
-            mock_file_open.side_effect = mock_open
+            mock_open.side_effect = mock_file_open
```

Comment:
I'm not sure this work. side effect means you are setting the return value. This is what you need http://www.voidspace.org.uk/python/mock/helpers.html#mock-open